src/App.js

```diff
@@ -0,0 +1,78 @@
+import React, { Fragment, useState, useEffect } from 'react';
+import coffeMilk from './imgs/coffeMilk.png';
+import coffe from './imgs/coffe.png';
+import sandwich from './imgs/sandwich.png';
+import juice from './imgs/juice.png';
+import simpleBurguer from './imgs/simpleBurguer.png';
+import doubleBurguer from './imgs/doubleBurguer.png';
+import frenchFries from './imgs/frenchFries.png';
+import onion from './imgs/onion.png';
+import smallWater from './imgs/smallWater.png';
+import bigWater from './imgs/bigWater.png';
+import smallSoda from './imgs/smallSoda.png';
+import bigSoda from './imgs/bigSoda.png';
+
+
+import { db } from './firebase/firebase-config'
+import Routes from './components/Routes.js';
```

Comment:
```suggestion
import React, { Fragment, useState, useEffect } from 'react';
import { db } from './firebase/firebase-config'
import Routes from './components/Routes.js';

import coffeMilk from './imgs/coffeMilk.png';
import coffe from './imgs/coffe.png';
import sandwich from './imgs/sandwich.png';
import juice from './imgs/juice.png';
import simpleBurguer from './imgs/simpleBurguer.png';
import doubleBurguer from './imgs/doubleBurguer.png';
import frenchFries from './imgs/frenchFries.png';
import onion from './imgs/onion.png';
import smallWater from './imgs/smallWater.png';
import bigWater from './imgs/bigWater.png';
import smallSoda from './imgs/smallSoda.png';
import bigSoda from './imgs/bigSoda.png';

```

Primero las dependencias